### PR TITLE
bug/#2681-program-entity-summary-section-with-multiple-values-missing-delimiter(comma)

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
**Issue:** Program Entity table rows with multiple values are missing spaces and comma between these values. These rows are regions, countries, institutions, primarySites and cancerTypes

**Solution:** format the query with array.join() method. The results are values with space and commas in between.

Was able to test countries, primarySites in dev. Couldn't edit programs to test the other three rows. But since the update works on countries and primarySites, it should work for the rest as well.

**Screenshot**
Showing space and comma between values in primarySites and countries.
Screenshot 2023-11-30 at 4 20 09 PM

## Type of Change

- [x] Bug
- [ ] Dependency updates
- [ ] Feature
- [ ] Refactoring
- [ ] Release candidate
- [ ] Styling
- [ ] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [x] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [ ] No back-end changes
- [x] Manually tested changes
- [ ] Added copyrights to new files
- [x] Connected ticket to PR

### CODE REVIEW

File: .husky/pre-commit
Comment (demariadaniel) [inline .husky/pre-commit:1]: Why are these deleted? I've had them interfere with my local dev before, but as I remember we still want them in the repo

Comment (daniel-cy-lu) [reply .husky/pre-commit:1]: Hi Dan, we are not deleting, but adding space and comma between the row that has multiple values. Right now there is no space between them. See the screenshot in this ticket: https://app.zenhub.com/workspaces/icgc-argo-platform-dk-production-board-5e542d38415f5034e9fed89d/issues/gh/icgc-argo/platform-ui/2681

Comment (demariadaniel) [reply .husky/pre-commit:1]: The code for adding commas looks fine, but the PR contains commits which delete our Husky pre-commit / pre-push files

Comment (demariadaniel) [reply .husky/pre-commit:1]: [2c49136ea](https://github.com/icgc-argo/platform-ui/pull/2745/commits/2c49136ea48d7a3c57b95fecca88c64dea79e355)

Comment (daniel-cy-lu) [reply .husky/pre-commit:1]: ohh right, I am not sure what to do with this. I was playing around because I couldn't push this ticket. let me see if I could fix it. Sorry I forgot to address this.

Comment (demariadaniel) [reply .husky/pre-commit:1]: @samrichca last discussed this with you, do you have any suggestions how we can get around this going forward?

Comment (daniel-cy-lu) [reply .husky/pre-commit:1]: @demariadaniel I added back the husky folder and was able to git push this time. I don't the reasoning behind this though.

**Review by demariadaniel — APPROVED** [review]

**Comment by daniel-cy-lu** [comment]
@demariadaniel Thank you Dan like always!
